SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -13,33 +13,52 @@ In order to interact with each other, Unreal and non-Unreal server-worker instan
 
 Default single Unreal server-worker type development doesn’t accommodate schema files that haven’t been generated by the GDK, so you need to set your game up to handle this.
 
+>**TIP:** If you’re using schema from outside the GDK, you can customise [snapshot generation](https://docs.improbable.io/unreal/alpha/content/generating-a-snapshot#generating-a-snapshot) from the GDK toolbar’s  **Snapshot** button. This is to serialize additional entities with these external components which the default Unreal snapshot generation cannot currently do. See [Add to the snapshot](#add-to-the-snapshot) below for how to do this.
```

Comment:
Could you use `{{urlRoot}}` for the snapshot generation link to prevent it from being broken in the future?